src/developer-release-signer.md

```diff
@@ -0,0 +1,5 @@
+# Developer Release
+
+[The sBTC Developer Release](./sbtc-releases/sbtc-dev.md) is a minimum-viable testnet deployment of sBTC and was decided to be released without the without the complexity of adding a decentralized set of signers to validate sBTC deposits/withdrawals.
+
+Therefore our beloved the Developer Release aka Version 0.1 Alpha Romeo won't have signer when deployed to public test net.
```

Comment:
Therefore our beloved Developer Release aka Version 0.1 Alpha Romeo only has one single signer that is operated by a central authority.

Reply:
fixed cbec4022241ad010c47e210512c1112d94883e68